File: app/src/main/java/com/nimtego/plectrum/presentation/main/model/AlbumModel.java
package com.nimtego.plectrum.presentation.main.model;

import lombok.Builder;
import lombok.Data;
import lombok.ToString;

@Data
@Builder
@ToString
@Deprecated
public class AlbumModel {
    private String albumName;
    private String albumArtistName;
    private String albumArtwork;
    private String albumId;
    private String albumArtWorkUrl;

}

File: app/src/main/java/com/nimtego/plectrum/presentation/mvp/model/music/AlbumModel.kt
package com.nimtego.plectrum.presentation.mvp.model.music

data class AlbumModel(
        val albumName: String,
        val albumArtistName: String,
        val albumArtwork: String,
        val albumId: String,
        val albumPrice: String,
        val albumReleaseDate: String,
        val albumSongs: List<SongModel> = emptyList())
